database/01_table_creation.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE token_info (
```

Comment:
Shouldn't we make it chain-specific? I would rename it to

`token_info_mainnet`

given that we will probably use one version of the db for all chains

Reply:
Why would we have one database for _all_ chains instead of one database _per_ chain? We use the latter for all other purposes.

If we go with having information on all chain in one database, I would expect that having an additional column `network` in all tables is better that duplicating tables for each chain.

Reply:
> Why would we have one database for all chains instead of one database per chain? We use the latter for all other purposes.

Don't have a strong opinion here, but this would add overhead, as we (i.e., devops) would need to set up different databases, accounts etc

> If we go with having information on all chain in one database, I would expect that having an additional column network in all tables is better that duplicating tables for each chain.

Although postgres most likely takes care of everything, having a single table where multiple daemons might try to add things at the same time could create some race conditions. Also, could it make things slower, as network should be part of the primary key now?

Somehow having a separate table per chain looks safer/more flexible.

Reply:
What do you think, @ahhda?